index.html

```diff
@@ -34,12 +34,163 @@ <h1>I’m Omayio Glad to see you!</h1>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
+        <div class="grid-item show-case4">
+          <div class="content">
+            <h4>Multi-Post Stories</h4>
+            <ul class="flex">
+              <li class="cano">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
       </section>
+      <section id="about-me" class="flex">
+        <h2>About Myself</h2>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h5>LET’S CONNECT</h5>
+        <ul class="flex">
+          <li><a href="#"><img src="imgs/Github.png" alt="Github Icon" /></a></li>
+          <li><a href="#"><img src="imgs/Linkedin-icon.png" alt="Linkedin-icon" /></a></li>
+          <li><a href="#"><img src="imgs/hands.png" alt="hands" /></a></li>
+          <li><a href="#"><img src="imgs/Twitter.png" alt="Twitter Icon" /></a></li>
+          <li><a href="#"><img src="imgs/M.png" alt="M icon" /></a></li>
+        </ul><br><br>
+        <button type="submit">Get My Resume</button>
+        <ul class="lang-section">
+          <li>
+            Languages
+            <img src="imgs/Icon-Down.png" alt="#" />
+          </li>
+        </ul>
+        <div class="lang-prog">
+          <img class="javascript-img" src="imgs/JS.png" alt="Javascript">
+          <span> &nbsp; &nbsp; JavaScript</span>
+        </div>
+
+        <div class="lang-prog">
+          <img class="html-img" src="imgs/HTML.png" alt="HTML">
+          <span> &nbsp; &nbsp; HTML</span>
+        </div>
+
+        <div class="lang-prog">
+          <img class="css-img" src="imgs/CSS.png" alt="css">
+          <span> &nbsp; &nbsp; JavaScript</span>
+        </div>
+
+
+        <ul class="lang-section hg">
+          <li>
+            Frameworks
+            <img src="imgs/Icon-Right.png" alt="" />
+          </li>
+          <li>
+            Skills <img src="imgs/Icon-Right.png" alt="" />
+          </li>
+        </ul>
+      </section>
+
+    </div>
+
+    <footer>
```

Comment:
- [ ] The request from the previous reviewer was to add padding at the top of the footer section for better look and not to remove the background image. Therefore, you should add the `contact form shapes-mobile` background image as well from the Figma design. Please also make sure that you have not spaces in the image file name because images with file names that has spaces might not be rendered correctly.
![contact-form-bg-missing-impl](https://user-images.githubusercontent.com/44057863/165687223-b7f4d76c-121b-4d44-8c1f-5d3335c8bb95.png)

Reply:
@gtekle , I could view the image from my PC. I genuinely cant understand why you couldnt. However, i have renamed the name of my background image from 'contact-form shapes-mobile.png' to 'contact_form_shapes_mobile.png'.
I used escape character ( \ ) to let CSS ignore the space during my previous commit.
I hope the current reviewer will be able to view the image :disappointed: 

![image](https://user-images.githubusercontent.com/93031593/165735641-8c70fb3c-98e0-41ac-9869-aa92fcf908fe.png)